### PR TITLE
Docs: Reword the HyperFormula license note

### PR DESCRIPTION
**Pull request by kirszenbaum**
As discussed [here](https://handsoncode.slack.com/archives/C031GBX62SE/p1671539765686289), this PR changes the wording of the HyperFormula license note.

[skip changelog]

### CODE REVIEW

**Review by AMBudnik — APPROVED** [review]
👍

File: docs/content/guides/formulas/formula-calculation.md [hunk 1]
```diff
@@ -56,9 +56,9 @@ To find out which HyperFormula version to use, see the table below:
 | [`12.x.x`](https://github.com/handsontable/handsontable/releases/tag/12.0.0) and higher | [`^2.0.0`](https://github.com/handsontable/hyperformula/releases/tag/2.0.0)                                    |
 
 ::: tip
-Your [Handsontable license key](@/guides/getting-started/license-key.md) lets you use HyperFormula for free in your Handsontable instance: just pass `'internal-use-in-handsontable'` as your HyperFormula license key .<br><br>
+You can use the `'internal-use-in-handsontable'` license key only in those HyperFormula instances that are connected to a Handsontable instance.
```
Comment (sequba) [inline docs/content/guides/formulas/formula-calculation.md:59]: Is `<br><br>` not needed anymore?

Comment (sequba) [inline docs/content/guides/formulas/formula-calculation.md:59]: Is `<br><br>` not needed anymore?

Comment (kirszenbaum) [reply docs/content/guides/formulas/formula-calculation.md:59]: No, I think it shouldn't have been there in the first place, the note looks good in the output:

Comment (sequba) [reply docs/content/guides/formulas/formula-calculation.md:59]: ok

**Review by sequba — APPROVED** [review]

**Pull request merged**